### DISCUSSION TITLE
New update for ASUS System Control Interface v3 (Version V3.1.29.0)

### DISCUSSION THREAD

**Discussion by xanver999** [seerge/g-helper#2737] [Q&A] [answered]
Should I update it or will it mess up with Ghelper? Anyone tested it?

**Comment by Kikimahe** [comment, marked as answer]
Hi, 

I installed it and no problem with Ghelper.

**Reply by xanver999** [reply]
thanks. I'll update it then.

**Comment by xSimone** [comment]
What does this do specifically?
